src/core/geometry/qgsgeometryutils.h

```diff
@@ -291,6 +291,31 @@ class CORE_EXPORT QgsGeometryUtils
      */
     static QgsPointV2 midpoint( const QgsPointV2& pt1, const QgsPointV2& pt2 );
 
+    /** Return the slope of a line defined by points pt1 and pt2.
```

Comment:
Small style change, but we're moving away from explicit listing of all params in cases like this (where the meaning has already been described). It's an nice touch to change "defined by points P1 and p2" to "defined by points \a p1 and \a p2" so that doxygen formats these as b arguments.